src/framapp-polymer-app/framapp-polymer-app.html

```diff
@@ -47,10 +47,7 @@
                  pattern="/:page"
                  data="{{routeData}}"
                  tail="{{subroute}}"></app-route>
-      <iron-ajax auto
-                 url="https://project-5613440220430148247.firebaseio.com/api/v1/categories.json"
-                 handle-as="json"
-                 last-response="{{categories}}"></iron-ajax>
+      <fp-data-store source-url="https://project-5613440220430148247.firebaseio.com/api/v1/categories.json" categories={{categories}}></fp-data-store>
```

Comment:
I don't think that `sourceUrl` property is good design. The `fp-data-store` should *know* where to get its data from. When we get there we can externalize the firebase URL to some share state/setting